README.md

```diff
@@ -1 +1,87 @@
-# shelf-lib-js
\ No newline at end of file
+shelf-lib-js
+============
+
+
+Introduction
+------------
+
+shelf-lib-js is a Node.js library for using pyshelf[https://github.com/kyle-long/pyshelf].
+
+
+Installation
+------------
```

Comment:
Could you separate the description from the code example?  Just have another subheading.